File: socketd/src/main/java/org/noear/socketd/exception/SocketdException.java
package org.noear.socketd.exception;

/**
 * @author noear
 * @since 2.0
 */
public class SocketdException extends RuntimeException {
    public SocketdException(Throwable cause){
        super(cause);
    }

    public SocketdException(String message){
        super(message);
    }
}

File: dotnet/socketd/org.noear.socketd/exception/SocketdException.cs
namespace org.noear.socketd.exception;

/**
 * 异常
 *
 * @author noear
 * @since 2.0
 */
public class SocketdException : Exception
{
    public SocketdException(string message) : base(message)
    {

    }

    public SocketdException(string message, Exception cause) : base(message, cause)
    {

    }
}